components/http_client.rst

```diff
@@ -745,20 +745,15 @@ HTTPlug
 The `HTTPlug`_ specification was published before PSR-18 and is superseded by
 it. As such, you should not use it in newly written code. Yet, many libraries
 still require v1 or v2 of it. The component is interoperable with them thanks to
-the ``HttplugClient`` adapter class. Similarly to ``Psr18Client`` implementing
-relevant parts of PSR-17, ``HttplugClient`` also implements the factory methods
-defined in the related ``php-http/message-factory`` package.
```

Comment:
I did not change these 3 lines. I just added a class reference to `HttplugClient`